src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int SIZE_INDEX = 2;
+    private Node<K, V>[] listOfPair;
+    private int size;
+    private int treshold;
```

Comment:
there is no need for this variable, you can calculate it through size, but let it stay :)